signinapp/templates/event_register.html.jinja2

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html.jinja2" %}
```

Comment:
No way to get to this page?

Reply:
Yeah, I was thinking of adding a new event page that students can see enabled events at, though I need to see how we're using that field. Or we could just rely upon a mentor sharing the link which is likely how students will see it anyway.